docs/resources/flow.md

```diff
@@ -72,4 +72,6 @@ resource "genesyscloud_flow" "flow" {
 ### Read-Only
 
 - `id` (String) The ID of this resource.
+- `name` (String) Flow name.
+- `type` (String) Flow type. Valid options: bot, commonmodule, digitalbot, inboundcall, inboundchat, inboundemail, inboundshortmessage, outboundcall, inqueuecall, inqueueemail, inqueueshortmessage, speech, securecall, surveyinvite, voice, voicemail, voicesurvey, workflow, workitem
```

Comment:
there is another comment from Adam. https://inindca.atlassian.net/browse/DEVTOOLING-854?focusedCommentId=5790355 to extend this to datasource too. but looks like datasource already have it in its schema. just reverify if you are able to export datasource with filename and type intact. ( use replacewithdatasource option for export)

Reply:
Verified